src/main/java/mate/controller/IndexController.java

```diff
@@ -0,0 +1,15 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class IndexController extends HttpServlet {
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("webinf/views/index.jsp").forward(req,resp);
```

Comment:
```suggestion
        req.getRequestDispatcher("WEB-INF/views/index.jsp").forward(req,resp);
```
next templates will use this naming. also, the standart is `WEB-INF`, `META-INF` etc